libs/MVS/SceneTexture.cpp

```diff
@@ -151,24 +151,45 @@ struct MeshTexture {
 		typedef TRasterMesh<RasterMesh> Base;
 		FaceMap& faceMap;
 		FIndex idxFace;
+		cv::Mat mask;
```

Comment:
use out improvements of cv::Mat for images, you will see it is easier to use:
here use Image8u as you only need an image with one unsigned byte channel